src/lib/EventSubscriber/RestKernelViewSubscriber.php

```diff
@@ -1,111 +0,0 @@
-<?php
-
-/**
- * @copyright Copyright (C) Ibexa AS. All rights reserved.
- * @license For full copyright and license information view LICENSE file distributed with this source code.
- */
-
-namespace Ibexa\HttpCache\EventSubscriber;
-
-use FOS\HttpCache\ResponseTagger;
-use Ibexa\Contracts\Core\Repository\Values\Content\Section;
-use Ibexa\Contracts\Core\Repository\Values\ContentType\ContentType;
-use Ibexa\Contracts\HttpCache\Handler\ContentTagInterface;
-use Ibexa\Rest\Server\Values\CachedValue;
-use Ibexa\Rest\Server\Values\ContentTypeGroupList;
-use Ibexa\Rest\Server\Values\ContentTypeGroupRefList;
-use Ibexa\Rest\Server\Values\RestContentType;
-use Ibexa\Rest\Server\Values\VersionList;
-use Ibexa\Rest\Values\Root;
-use Symfony\Component\EventDispatcher\EventSubscriberInterface;
-use Symfony\Component\HttpKernel\Event\ViewEvent;
-use Symfony\Component\HttpKernel\KernelEvents;
-
-/**
- * Set cache tags on a few REST responses used by UI in order to be able to cache them.
- *
- * @deprecated This is a temprary approach to caching certain parts of REST used by UI, it is deprecated in favour of
- *             nativly using tags and CachedValue in kernel's REST server itself once we switch to FOSHttpCache 2.x
- *             where tagger service can be used directly.
```

Comment:
Internal sync: seems there's more work to be done to utilize FOSHttpCache than just removing the subscriber.